docs/migration_guide.rst

```diff
@@ -5,22 +5,69 @@ Migration guide
 0.19.0 Migration guide
 **********************
 
+Version 0.19.0 of **starknet.py** comes with support for RPC 0.6.0!
+
+.. currentmodule:: starknet_py.net.client_models
+
+New classes added to mirror the recent changes in the RPC v0.6.0 specification include:
+:class:`ResourceBoundsMapping`, :class:`ResourceBounds`, :class:`PriceUnit`, :class:`FeePayment`, :class:`DAMode`.
+
+Changes in the :class:`~starknet_py.net.account.account.Account`:
+
+.. currentmodule:: starknet_py.net.account.account
+
+- :meth:`~Account.execute_v3` has been added.
+- :meth:`~Account.sign_declare_v3_transaction`, :meth:`~Account.sign_deploy_account_v3_transaction` and :meth:`~Account.sign_invoke_v3_transaction` have been added.
+- :meth:`~Account.sign_declare_transaction`, :meth:`~Account.sign_deploy_account_transaction` and :meth:`~Account.sign_invoke_transaction` have been renamed to :meth:`~Account.sign_declare_v1_transaction`, :meth:`~Account.sign_deploy_account_v1_transaction` and :meth:`~Account.sign_invoke_v1_transaction` respectively.
+
+All new functions with ``v3`` in their name operate similarly to their ``v1`` and ``v2`` counterparts.
+The key difference resides in the parameter acceptance: they don't accept the ``max_fee`` parameter. Instead, they utilize the ``l1_resource_bounds`` parameter.
+
+Changes in the :class:`~starknet_py.net.full_node_client.FullNodeClient`:
+
+.. currentmodule:: starknet_py.net.full_node_client
+
+- :meth:`~FullNodeClient.estimate_fee` has a new parameter ``skip_validate``.
+- :meth:`~FullNodeClient.declare` accepts ``transaction`` argument of the type :class:`~starknet_py.net.models.transaction.DeclareV3`
+- :meth:`~FullNodeClient.send_transaction` accepts ``transaction`` argument of the type :class:`~starknet_py.net.models.transaction.InvokeV3`
+- :meth:`~FullNodeClient.deploy_account` accepts ``transaction`` argument of the type :class:`~starknet_py.net.models.transaction.DeployAccountV3`
+
+.. note::
+    :class:`~starknet_py.contract.Contract` class does not support V3 transactions in the pre-release.
+
+
+0.19.0 Targeted versions
+------------------------
+
+- Starknet - `0.13.0 <https://docs.starknet.io/documentation/starknet_versions/version_notes/#version0.13.0>`_
+- RPC - `0.6.0 <https://github.com/starkware-libs/starknet-specs/releases/tag/v0.6.0>`_
+
 0.19.0 Breaking changes
```

Comment:
I think `ResourcePrice.price_in_strk` -> `ResourcePrice.price_in_fri` (with later field becoming required) change wasn't mentioned.
Let's make sure we go through #1243 and check if we included all of the breaking changes.